src/v/redpanda/application.cc

```diff
@@ -650,6 +651,10 @@ void application::wire_up_redpanda_services() {
       .get();
     vlog(_log.info, "Partition manager started");
 
+    if (coproc_enabled()) {
+        _wasm_event_listener = std::make_unique<coproc::wasm::event_listener>();
```

Comment:
I wouldn't move this outside of coproc. If you need a handle to this just do something like:
```
namespace coproc {
class api {
public:
  wasm::event_listener* get_event_listener() { return _listener.get() }
  ...
  ...
};
}
```
And call the method to grab an un-owned reference to this class

Reply:
But v8_api needs event_listener and coporc can be disabled in that moment

Reply:
I think data_policy and coproc are independent features